include/fmt/format.h

```diff
@@ -644,11 +636,20 @@ FMT_CONSTEXPR bool is_negative(T) {
   return false;
 }
 
+#if FMT_USE_INT128
+// Smallest of uint32_t, uint64_t, unsigned __int128 that is large enough to represent all
```

Comment:
nit: This seems like a long line. Could you apply clang-format to your changes or reflow the comment to fit into 80 cols?